src/reynir_correct/errtokenizer.py

```diff
@@ -2800,7 +2831,15 @@ def check_taboo_words(token_stream: Iterable[CorrectToken]) -> Iterator[CorrectT
 
     for token in token_stream:
         # Check taboo words
-        if token.has_meanings:
+        if (  # type: ignore
+            token.val is not None
+            and token.has_meanings
+            and token.txt not in NOT_TABOO
```

Comment:
This does not work if there are underscores in the NOT_TABOO strings, such as "hommi_kk"

Reply:
...and besides, we probably want to match all forms of the lemma, i.e. "hommar", "hommans", etc. so this should be a comparison with .stofn, right?